tarxx.h

```diff
@@ -53,14 +53,17 @@
 
 namespace tarxx {
 
+using std::string_literals::operator""s;
+
 #ifdef __linux
     struct errno_exception : std::system_error {
         errno_exception() : std::system_error(std::error_code(errno, std::generic_category())) {}
         using std::system_error::system_error;
     };
 #endif
 
-    using block_t = std::array<char, 512>;
+    static constexpr const int BLOCK_SIZE = 512;
```

Comment:
`constexpr` implies `const`, please remove the `const`.
```suggestion
    static constexpr int BLOCK_SIZE = 512;
```

Reply:
Removed